.github/workflows/build.yaml

```
@@ -0,0 +1,17 @@
+name: Docker build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  docker-build:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
+    with:
+      name: runner-manager
+      namespace: scality
```

There are other ways to achieve this in case this one doesn't work but when building on the main branch, tag the image to `latest`
```suggestion
      namespace: scality
      tag: ${{ github.ref_name == 'main' && 'latest' || github.sha }}
```